_books/ion-1-1/src/binary/values/symbol.md

```diff
@@ -61,3 +61,22 @@ address that is decoded is biased by the number of addresses that can be encoded
 | `0xE1` | 0 to 255             | 0      |
 | `0xE2` | 256 to 65,791        | 256    |
 | `0xE3` | 65,792 to infinity   | 65,792 |
+
+
+### System Symbols
+
+<!-- TODO: Add link to "system-module" page somewhere in this section. /-->
+
+System symbols (that is, symbols defined in the system module) can be encoded using the `0xEF` opcode followed by a _negative_ 1-byte `FixedInt`.
+(Positive values are used for [system macro invocations](../e_expressions.md#system-macro-invocations).)
+
+Unlike Ion 1.0, symbols are not required to use the lowest available SID for a given text.
+System symbols _MAY_ be encoded using other SIDs, but if an Ion data stream uses imported SIDs for system symbols in encoding directives, then the data may be inaccessible.
```

Comment:
The data is "all of your data".

Suppose you import some symbols, and `$123` is `$ion_encoding`, and it is imported from shared module Foo.
Then, you want to append some symbols, and you do this:
(There's some potential pseudo-code going on here, so don't get to concerned about the syntax.)
```
$123::(
    (import $123)
    (symbols $123 ["cat", "dog"])
)

// Lots more user data
```
If `Foo` is not available, then you can't even read the encoding directive here because you don't know that `$123` is `$ion_encoding`.

Reply:
But I suppose that's no different than a macro being unavailable for a top-level e-expression.